platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1363,10 +1361,12 @@ public void onSurfaceCreated(final GL10 gl, final EGLConfig config) {
     // ==================
     @Keep
     void cancelUrlRequest(final long requestHandle) {
-        if (httpHandler == null) {
-            return;
+        if (httpHandler != null) {
+            Object request = httpRequestHandles.get(requestHandle);
+            if (request != null) {
```

Comment:
Hmm personally I'd leave the request objects as-is and let the implementation deal with any nulls that it created for itself. But I'm not strongly opposed to this either.

Reply:
I would like to have this safety net here.